### DISCUSSION TITLE
Watchdog

### DISCUSSION THREAD

**Discussion by mauroForlimpopoli** [Moddable-OpenSource/moddable#729] [Q&A] [answered]
Good morning, 
I'm trying to add a watchdog in my project. 
It work, but when it occurs I just only get 

```
abort() was called at PC 0x40176a1f on core 1
Backtrace:0x40091362:0x3ffbed28 |<-CORRUPTED
ELF file SHA256: eb2d303fba479f95
Entering gdb stub now.
abort() was called at PC 0x40176a1f on core 1
Backtrace:0x40091362:0x3ffbed28 |<-CORRUPTED
ELF file SHA256: eb2d303fba479f95
Entering gdb stub now.
```
on the command promt, but the device don't restart. This if is not in debug mode also.
How can I to restart the device ?
Regards

**Comment by phoddie** [comment, marked as answer]
You have the GDB stub enabled. If that is present, when the watchdog triggers it tries to go to GDB. That takes priority over restart.  The [SDKCONFIG for ESP32 release builds](https://github.com/Moddable-OpenSource/moddable/blob/public/build/devices/esp32/xsProj-esp32/sdkconfig.defaults.release) explicitly disables the GDB stub.

**Reply by mauroForlimpopoli** [reply]
I changed the value you told me in the file sdkconfig.defaults and now it works 
Thank you.
